From 8af9e371c1dca38813c422c75c1db404ad04f376 Mon Sep 17 00:00:00 2001
From: Lin Fuzhen <fuzhen.lin@freescale.com>
Date: Fri, 6 Apr 2012 10:43:42 +0800
Subject: [PATCH 199/249] ENGR00179050 VPU: Add MX6DL support in VPU library
 code

the orignal code just support MX6Q, add MX6DL support in VPU library
code.

Signed-off-by: Lin Fuzhen <fuzhen.lin@freescale.com>
---
 vpu/vpu_reg.h  | 2 +-
 vpu/vpu_util.h | 3 ++-
 2 files changed, 3 insertions(+), 2 deletions(-)

diff --git a/vpu/vpu_reg.h b/vpu/vpu_reg.h
index e0e87bc..760bfd3 100644
--- a/vpu/vpu_reg.h
+++ b/vpu/vpu_reg.h
@@ -396,7 +396,7 @@
         #define TEMP_BUF_SIZE			(304 * 1024)
         #define PARA_BUF2_SIZE			(2 * 1024)
         #define PARA_BUF_SIZE			(10 * 1024)
-#elif defined(IMX6Q)
+#elif defined(IMX6Q) || defined(IMX6DL)
 	#define CODE_BUF_SIZE                   (280 * 1024) /* define max is 280k byte currently */
 	#define FMO_SLICE_SAVE_BUF_SIZE         (32)
 	#define TEMP_BUF_SIZE                   (204 * 1024)
diff --git a/vpu/vpu_util.h b/vpu/vpu_util.h
index d511087..4dfd0e1 100644
--- a/vpu/vpu_util.h
+++ b/vpu/vpu_util.h
@@ -100,7 +100,8 @@ enum {
 	AVS_DEC = 0x81,
 	VPX_DEC = 0x82
 } ;
-#elif defined(IMX6Q)
+#elif defined(IMX6Q) || defined(IMX6DL)
+
 enum {
 	AVC_DEC = 0,
 	VC1_DEC = 1,
-- 
1.8.0

